snekbox/__main__.py

```diff
@@ -13,10 +13,16 @@ def parse_args() -> argparse.Namespace:
     )
     parser.add_argument("code", help="the Python code to evaluate")
     parser.add_argument(
-        "nsjail_args", nargs="?", default=[], help="override configured NsJail options"
+        "nsjail_args",
```

Comment:
Do you have a sample of what the help output is after this change?

Reply:
Yea, the reason I added the default to the help directly is because it didn't get added automatically.

```
❯ python3 -m snekbox --help
usage: snekbox [-h] code [nsjail_args ...] [--- py_args ...]

positional arguments:
  code         the Python code to evaluate
  nsjail_args  override configured NsJail options (default: [])
  py_args      arguments to pass to the Python process (default: ['-c'])

options:
  -h, --help   show this help message and exit
```

Reply:
Thanks - that still looks helpful